portality/models/preservation.py

```diff
@@ -54,4 +64,34 @@ def failed(self, message, details=None):
         self.status = "failed"
         self.data["error"] = message
         if details is not None:
-            self.data["error_details"] = details
\ No newline at end of file
+            self.data["error_details"] = details
+
+    @classmethod
+    def by_owner(cls, owner, size=10):
+        q = OwnerFileQuery(owner)
+        res = cls.query(q=q.query())
+        rs = [PreservationState(**r.get("_source")) for r in res.get("hits", {}).get("hits", [])]
+        return rs
+
+
+class OwnerFileQuery(object):
+    base_query = {
+        "query": {
+            "bool": {
+                "must": []
+            }
+        },
+        "sort": [
+            {"created_date": "desc"}
+        ],
+        "size": 10
+    }
+
+    def __init__(self, owner, size=10):
+        self._query = deepcopy(self.base_query)
+        owner_term = {"match": {"owner": owner}}
```

Comment:
I think here we'll need to do `owner.exact` as the field to match on.  Probably it won't make any difference, because owner fields are pretty simple, but in this query the field is the analysed field, so it's possible the match won't be exact.